wp-content/themes/engage/assets/scss/components/_research.scss

```diff
@@ -16,6 +16,32 @@
 	}
 }
 
+.aspect-ratio-box {
+	height: 0;
+	padding-top: 650px / 1127.34px * 100%;
+	position: relative;
+}
+
+.aspect-ratio-box-inside {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+}
+
+.article__video {
```

Comment:
I think `.article__video` could go with the article css, as it could really be expanded to any article whenever we wanted to.